cmd/main.go

```diff
@@ -34,7 +33,14 @@ const (
 	BROADCAST = "broadcast-trusted-state"
 )
 
+const (
+	//envCommitHash environment variable name for COMMIT_HASH
+	envCommitHash = "COMMIT_HASH"
+)
+
 var (
+	// commit represents the program based on the git commit
+	commit         = testutils.GetEnv(envCommitHash, "dev")
```

Comment:
@Psykepro this `GetEnv` looks useful even outside tests. WDYT of moving it outside the `testutils` (maybe in a `utils` package)?.
This is not directly related to this PR (and maybe it can involve other utils functions) so feel free to resolve this comment and keep it for later.
CC @KonradIT

Reply:
Yep, I was thinking of that because we have several functions in the `test` package which we use in other files like for example also the functions from `test/operations` outside of the test context. But as we discussed with @tclemos it is not good to have a too generic package like `utils` because it can get too big when everyone puts something there. But maybe we can have a package `environment` where to put this `GetEnv` function specifically. What do you think guys? @tclemos @kind84 @KonradIT @ARR552

Reply:
It can stay as it is. Getting value from env is only used in testing scenarios (be it E2E tests or standalone tests), so it makes sense the code stays within the `test` folder. Same with `dbutils`, etc...

Reply:
@Psykepro l close this thread and we can continue the discussion elsewhere as it seems to be a broader topic.